web/pages/notifications.tsx

```diff
@@ -56,46 +62,71 @@ export default function Notifications() {
   return (
     <Page>
       <div className={'px-2 pt-4 sm:px-4 lg:pt-0'}>
-        <Title text={'Notifications'} className={'hidden md:block'} />
+        <Row
+          className={clsx('w-full justify-between', isMobile ? 'hidden' : '')}
+        >
+          <Title text={'Notifications'} className="grow" />
+          <div className="my-auto">
+            {!isMobile && activeIndex === 0 && (
+              <MarkAllAsReadButton
```

Comment:
Hmmmm I am pretty unsure that we need this functionality. I think it's pretty good that notifications are assigned as read on reaching the notification page. This mirrors how twitter's notifications work. I get dozens of notifications per day and I do not want to have to click anything to mark them as read.